src/dsp_tools/utils/xmlupload/list_client.py

```diff
@@ -98,7 +98,8 @@ def _get_list_from_server(con: Connection, list_iri: str) -> List:
     list_info = list_object["listinfo"]
     children: list[dict[str, Any]] = list_object["children"]
     root_iri = list_info["id"]
-    list_name = list_info["name"]
+    # if the root node does not have a name, use the label instead
```

Comment:
yes, according to the API docs, that is the case. And I actually only realized that when tests failed for me, because in the anything test data we have precisely this case.